src/App.IntegrationTests/App.IntegrationTestsRef.csproj

```diff
@@ -1,12 +1,14 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="5.0.13" />
+    <PackageReference Include="Altinn.Authorization.ABAC" Version="0.0.5" />
```

Comment:
Hmm, I can't see that the changes in the test project would require a dependency to ABAC. Why was this introduced?

Reply:
I think it was a project reference from the test project to the ABAC project that I changed to be a nuget reference as the ..\..\..\...csproj doesn't work when we moved the code to a new repo. But the we should see the deletion of the project refernce which I don't. I'll check.

Reply:
Seems like it's not needed, it compiled fine when removed.